admin_guide/install/quick_install.adoc

```diff
@@ -7,28 +7,76 @@
 :toc: macro
 :toc-title:
 
+== Overview
 The _quick installation_ method allows you to use an interactive CLI utility to
 install OpenShift across a set of hosts. The utility is a Python wrapper around
-the Ansible-based installation method currently described in the
-link:https://github.com/openshift/training[OpenShift Enterprise 3 Beta training
-materials].
+the Ansible-based installation process, available at
+https://install.openshift.com.
 
-[NOTE]
+Alternatively, you can use the link:manual_install.html[manual installation]
+method to manually configure and run the Ansible playbooks.
+
+== Prerequisites
+
+The installation utility allows you to install OpenShift
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#master[master]
+and
+link:../../architecture/infrastructure_components/kubernetes_infrastructure.html#node[node]
+hosts. By default, any hosts you designate as masters during the installation
+process will also be configured as nodes that are marked
+link:../../admin_guide/manage_nodes.html#marking-nodes-as-unschedulable-or-schedulable[unschedulable].
```

Comment:
Right now Masters have to run a Node so that they are part of the SDN and port proxying works.  Required for things like https://bugzilla.redhat.com/show_bug.cgi?id=1230483